### DISCUSSION TITLE
WhatsApp Cloud option not available when adding a WhatsApp inbox

### DISCUSSION THREAD

**Discussion by fnyamweya** [chatwoot/chatwoot#5774] [Self hosted] [answered]
I managed to upgrade chatwoot to version 2.10.0 but I can see the WhatsApp Cloud option while creating a whatsapp inbox.
I only see the Twilio and 360Dialog options.

![Screenshot from 2022-10-28 07-45-38](https://user-images.githubusercontent.com/57399460/198504796-4e314a3e-9217-44cb-895f-de1bada6a8b8.png)

Operating System: Ubuntu 20.04
Self Hosted Instance

**Comment by sojan-official** [comment, marked as answer]
@fnyamweya  could you try recompiling the assets and restarting chatwoot 
https://www.chatwoot.com/docs/self-hosted/deployment/linux-vm#if-precompile-fails
